app/views/shared/_checkbox_group.html.slim

```diff
@@ -8,7 +8,7 @@
     options,
     value_method,
     text_method,
-    hint_method = hint_method,
+    hint_method,
```

Comment:
Does this definitely work with the form builder?

Reply:
yes it does, tested it. you can have a look at the review app too in case you want to double check it